server/src/main/java/org/elasticsearch/indices/SystemIndexDescriptor.java

```diff
@@ -573,49 +525,6 @@ public String getMinimumMappingsVersionMessage(String cause) {
         );
     }
 
-    /**
-     * Gets a standardized message when the node contains a data or master node whose version is less
-     * than that of the minimum supported version of this descriptor and its prior descriptors.
-     *
-     * @param cause the action being attempted that triggered the check. Used in the error message.
-     * @return the standardized error message
-     */
-    @Deprecated
-    public String getMinimumNodeVersionMessage(String cause) {
-        Objects.requireNonNull(cause);
-        final Version actualMinimumVersion = priorSystemIndexDescriptors.isEmpty()
-            ? minimumNodeVersion
-            : priorSystemIndexDescriptors.get(priorSystemIndexDescriptors.size() - 1).minimumNodeVersion;
-        return String.format(
-            Locale.ROOT,
-            "[%s] failed - system index [%s] requires all data and master nodes to be at least version [%s]",
-            cause,
-            this.getPrimaryIndex(),
-            actualMinimumVersion
-        );
-    }
-
-    /**
-     * Finds the descriptor that can be used within this cluster, by comparing the supplied minimum
-     * node version to this descriptor's minimum version and the prior descriptors minimum version.
-     *
-     * @param version the lower node version in the cluster
```

Comment:
Nit: the javadoc in the remaining overload (`getDescriptorCompatibleWith(MappingsVersion version)` should be updated (it was copy/pasted): 
```
     * @param version the cluster minimum mapping version
     * @return <code>null</code> if the cluster minimum version is lower than the minimum version in this descriptor,
```